doc/stdlib/dune

```diff
@@ -0,0 +1,17 @@
+; This is an ad-hoc rule to easy the migration, it should be handled
+; natively by Dune in the future.
+(rule
+  (targets html)
+  (deps
+   (source_tree %{project_root}/theories)
+   (source_tree %{project_root}/plugins)
+   (:header %{project_root}/doc/common/styles/html/coqremote/header.html)
+   (:footer %{project_root}/doc/common/styles/html/coqremote/footer.html))
+ (action
+  (progn
+   (bash "mkdir -p html")
+   (bash "%{bin:coqdoc} -q -d html --with-header %{header} --with-footer %{footer} --multi-index --html -g -R %{project_root}/theories Coq -R %{project_root}/plugins Coq $(find %{project_root}/theories %{project_root}/plugins -name *.v)"))))
```

Comment:
Same here. It should be possible to do this without bash. Just bind the dependencies in theories and plugins to a name. If that doesn't work, I think we should have a bug report for that.

Reply:
If I bind the dependencies I'll get all the files in the tree, how can I filter just the `.v` ones?

I had to use bash here due to quoting issues when inlining the `find` call.

Reply:
I see, so actually, the feature that you need here is a recursive glob. Basically, you want to write:

```
(deps (coq_files (glob_files theories/**/*.v)))
```

I'm not entirely sure if we have a ticket for this, but the feature was mentioned a few times before. I'll look it up.

Reply:
Here's the ticket if anyone is curious: https://github.com/ocaml/dune/issues/1866

Reply:
Added a comment to update in the future, thanks!